### PR TITLE
Track multiple responses per API

### PR DESCRIPTION
**Pull request by delvedor**
Proposal: https://gist.github.com/delvedor/eccf0936048dda2ecf0708156a8a726c

Fixes: https://github.com/elastic/elasticsearch-specification/issues/928

### CODE REVIEW

File: specification/cluster/remote_info/ClusterRemoteInfoResponse.ts [hunk 1]
```diff
@@ -21,10 +21,11 @@ import { DictionaryResponseBase } from '@_types/Base'
 import { integer, long } from '@_types/Numeric'
 import { Time } from '@_types/Time'
 
-export class Response extends DictionaryResponseBase<
-  string,
-  ClusterRemoteInfo
-> {}
+export class Response {
+  '200': {
+    body: DictionaryResponseBase<string, ClusterRemoteInfo>
```
Comment (delvedor) [inline specification/cluster/remote_info/ClusterRemoteInfoResponse.ts:26]: Can we remove `DictionaryResponseBase` from the spec and always use `Dictionary`?

```suggestion
    body: Dictionary<string, ClusterRemoteInfo>
```

Comment (Anaethelion) [reply specification/cluster/remote_info/ClusterRemoteInfoResponse.ts:26]: If we are not extending anymore and we directly specify the body type in the property of the response we don't need the `DictionaryResponseBase` which is only there to be extended from.

In line with the rest of the changes I'd say yes, we can.

**Comment by Anaethelion** [comment]
LGTM

File: specification/cluster/remote_info/ClusterRemoteInfoResponse.ts (hunk 1 above)
Comment (stevejgordon) [reply specification/cluster/remote_info/ClusterRemoteInfoResponse.ts:26]: The .NET client will require such responses to derive from a `DictionaryResponseBase` base class which includes custom JSON formatting and a backing dictionary which derived type expose through a meaningful name. I don't think this change is too much a problem though as my generator could likely infer that for any responses where the body is a Dictionary type. I'm not currently generating these responses for the v8 alpha as they're on the long TODO list!

**Pull request merged**